src/NLog.WindowsIdentity/NLog.WindowsIdentity.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net45;net35;netstandard1.5;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net45;net35;netstandard1.5;netstandard2.0;net6.0-windows</TargetFrameworks>
```

Comment:
Excellent. I agree with @304NotModified That `net8-windows` should also be added, since deciding to go down this road. Please also change `net6.0-windows` to `net6-windows`

Reply:
Ok, will tomorrow add net 8 in as suggested.